staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/structuredmerge.go

```diff
@@ -72,30 +72,41 @@ func NewCRDStructuredMergeManager(typeConverter TypeConverter, objectConverter r
 	}, nil
 }
 
+func objectGVKNN(obj runtime.Object) string {
+	name := "<unknown>"
+	namespace := "<unknown>"
+	if accessor, err := meta.Accessor(obj); err == nil {
+		name = accessor.GetName()
+		namespace = accessor.GetNamespace()
+	}
+
+	return fmt.Sprintf("name=%v, namespace=%v, gvk=%v", name, namespace, obj.GetObjectKind().GroupVersionKind())
```

Comment:
I'd format it as `namespace/name;gvk`. Actually I don't know how to show the gvk but `namespace/name` is common and we should do that at least.

Reply:
Also consider putting the parens here in the format instead of everywhere that it is used?

Reply:
> Also consider putting the parens here in the format instead of everywhere that it is used?

I did try that first, and it didn't really look good.